### ISSUE TITLE
Upgrades for web map display (add min_zoom)

### ISSUE THREAD

**Issue by nvkelso** [nvkelso/natural-earth-vector#202]
**Commonly used now:**

_As used by Tilezen, Mapzen's vector tile stack._

- [x] ne_10m_admin_0_boundary_lines_land
    - [x] min_zoom
    - [x] min_zoom for the 10m polygons 
- [x] ne_10m_admin_0_boundary_lines_map_units
    - [x] min_zoom
    - [x] min_zoom for the 10m polygons 
- [x] ne_10m_admin_1_states_provinces_lines
    - [x] min_zoom
    - [x] min_zoom for the 10m polygons 
- [x] ne_10m_coastline
    - [x] min_zoom
- [x] ne_10m_lakes
    - [x] min_zoom
- [x] ne_10m_land
    - [x] min_zoom
- [x] ne_10m_ocean
    - [x] min_zoom
- [x] ne_10m_playas
    - [x] min_zoom
    - [x] character encoding funk
- [x] ne_10m_populated_places
    - [x] min_zoom
    - [x] character encoding funk
- [x] ne_10m_roads
    - [x] min_zoom
- [x] ne_10m_urban_areas
    - [x] min_zoom
- [x] ne_50m_admin_0_boundary_lines_land
    - [x] min_zoom
    - [x] min_label
    - [x] min_zoom for the 10m polygons 
- [x] ne_50m_admin_1_states_provinces_lines
    - [x] min_zoom
    - [x] min_label
    - [x] min_zoom for the 10m polygons 
- [x] ne_50m_coastline
    - [x] min_zoom
- [x] ne_50m_lakes
    - [x] min_zoom
    - [x] min_zoom for the 10m polygons 
- [x] ne_50m_land
    - [x] min_zoom
- [x] ne_50m_ocean
    - [x] min_zoom
- [x] ne_50m_playas
    - [x] min_zoom
- [x] ne_50m_urban_areas
    - [x] min_zoom
    - [x] min_zoom for the 10m polygons 
- [x] ne_110m_admin_0_boundary_lines_land
    - [x] min_zoom
    - [x] min_zoom for the 10m polygons 
- [x] ne_110m_coastline
    - [x] min_zoom
- [x] ne_110m_lakes
    - [x] min_zoom
    - [x] min_zoom for the 10m polygons 
- [x] ne_110m_land
    - [x] min_zoom
    - [x] oops: bad scalerank (to 0 from 1), bad featureclass (to Land from Country)
- [x] ne_110m_ocean
    - [x] min_zoom
  
**Should be used more often:**

- [x] ne_10m_geography_regions_points
    - [x] min_zoom basic
    - [x] character encoding funk
- [x] ne_10m_geography_regions_elevation_points
    - [x] min_zoom basic 0-7
    - [x] min_zoom zooms 8++
    - [x] character encoding funk (around 42 features, rescued from v1.4)
- [x] ne_10m_lakes_north_america
- [x] ne_10m_lakes_europe
    - [x] min_zoom
    - [x] character encoding funk (present since v1.2 release, problem with EU source?)
- [x] ne_10m_rivers_europe
    - [x] min_zoom
    - [x] character encoding funk
- [x] ne_10m_rivers_north_america
    - [x] min_zoom
- [x] ne_10m_minor_islands
    - [x] min_zoom
- [x] ne_10m_reefs
    - [x] min_zoom
- [x] ne_10m_rivers_lake_centerlines
    - [x] min_zoom
- [x] ne_10m_rivers_lake_centerlines_scale_rank
    - [x] min_zoom
- [x] ne_10m_geography_regions_polys
    - [x] min_zoom
- [x] ne_10m_geography_marine_polys
    - [x] min_zoom
- [x] ne_10m_admin_0_disputed_areas
    - [x] min_zoom
- [x] ne_10m_antarctic_ice_shelves_polys
    - [x] min_zoom
    - [x] character encoding funk
- [x] ne_10m_glaciated_areas
    - [x] min_zoom
    - [x] character encoding funk

**Earlier scale sets:**

- [x] ne_50m_rivers_lake_centerlines
    - [x] min_zoom
- [x] ne_110m_rivers_lake_centerlines
    - [x] min_zoom
- [x] ~~ne_50m_minor_islands~~ doesn't exist
- [x] ~~ne_110m_minor_islands~~ doesn't exist
- [x] ~~ne_50m_reefs~~ doesn't exist
- [x] ~~ne_110m_reefs~~ doesn't exist
- [x] ne_50m_geography_regions_points
    - [x] min_zoom
    - [x] character encoding funk?
- [x] ne_110m_geography_regions_points
    - [x] min_zoom
    - [x] ~~character encoding funk?~~ not an issue
- [x] ne_50m_geography_regions_elevation_points
    - [x] min_zoom
    - [x] ~~character encoding funk?~~ not an issue
- [x] ne_110m_geography_regions_elevation_points
    - [x] min_zoom
    - [x] ~~character encoding funk?~~ not an issue
- [x] ne_50m_geography_regions_polys
    - [x] min_zoom
- [x] ne_110m_geography_regions_polys
    - [x] min_zoom
- [x] ne_50m_geography_marine_polys
    - [x] min_zoom
- [x] ne_110m_geography_marine_polys
    - [x] min_zoom
    - [x] fix char encoding :(
- [x] ne_50m_admin_0_disputed_areas
    - [x] min_zoom
- [x] ~~ne_110m_admin_0_disputed_areas~~ doesn't exist
- [x] ne_50m_antarctic_ice_shelves_polys
    - [x] min_zoom (and fix scalerank to `0`)
    - [x] ~~character encoding funk~~ no names
- [x] ~~ne_110m_antarctic_ice_shelves_polys~~ doesn't exist
- [x] ne_50m_glaciated_areas
    - [x] min_zoom
    - [x] ~~character encoding funk~~ no names
- [x] ne_110m_glaciated_areas
    - [x] min_zoom
    - [x] ~~character encoding funk~~ no names

Other:

- [x] Copy over adm0 property changes to adm0 Excel file
- [x] Copy over adm0 min_zoom changes to scale_rank label points
- [x] Copy over adm1 property changes from lakes file to prime file and to scale_rank label points
- [x] Copy over land min_zoom changes to scale_rank label points
- [x] Copy over ocean min_zoom changes to scale_rank label points

**Comment by nvkelso** [comment]
Subtask includes: https://github.com/nvkelso/natural-earth-vector/issues/199

**Comment by nvkelso** [comment]
This is still WIP after #199 merge for mostly land and ocean themes in 1:10m, 1:50m, and 1:110m themes.

**Comment by nvkelso** [comment]
This remains the final outstanding task for the v4.0 milestone.

Once the min_zoom are copied over then the adm0 and adm1 themes will also need a rebuild using the scripts.

**Comment by nvkelso** [comment]
Ocean scale rank is a little odd – We only have 22 big features that get constructed that are "negative space". We'd need a separate build for each zoom I think if we wanted to account for the min_zoom of the coastline, which is out of scope for this release. Workaround is to build the water off the WGS84 bounding box (so the map background is "blue") and then overlay the land which has proper positive space ranks.

**Comment by nvkelso** [comment]
QA followups in #223.

**Comment by nvkelso** [comment]
(It's possible that additional themes should also have min_zoom values added – but that's for 4.1 milestone or later.)
